src/content/drafts/introducing-bulk-suppressions.md

```diff
@@ -0,0 +1,103 @@
+
+## Getting started
+
+Once you've configured a new rule as `"error"` in your ESLint configuration, you can bulk suppress existing violations using the command line. You can choose to suppress all violations or only those for specific rules.
+
+In either case, it's recommended to run the suppression command with the `--fix` flag to automatically resolve any violations that can be fixed.
+
+```
+# Suppress all violations
+eslint --suppress-all
+
+# Fix all violations and suppress them
+eslint --suppress-all --fix
+
+# Suppress specific rules
+eslint --suppress-rule <rule-name>
+```
+
+When you execute any of the above commands, ESLint will create a file called `eslint-suppressions.json` in the project root. This file contains the number of violations for each rule and file. The suppressions file is structured as follows:
+
+```json
+{
+  "src/file1.js": {
+    "no-undef": {
+        "count": 1
+    }
+  },
+  "src/file2.js": {
+    "no-unused-expressions": {
+        "count": 2
+    }
+  }
+}
+```
+
+ESLint will suppress all `"error"` violations for the specified rules in subsequent runs and will not report them. However, if more violations are found for the same rule in the same file, ESLint will report **all** of them. For example, if a file originally had 2 suppressed `no-undef` errors but now has 5, ESLint will display all 5 `no-undef` errors.
+
+This approach is intentional and helps avoid confusion. Because suppressions are tracked per file and rule—not per specific line or code change—there’s no reliable way to determine whether the new violations were introduced recently or already existed. Rather than trying to guess which violations to hide, ESLint chooses to show the full picture.
+
+### Changing the suppressions file location
+
+By default, ESLint creates the suppressions file as `eslint-suppressions.json` in the project root. You can use the `--suppressions-location` option to specify a different location for the suppressions file.
+
+The `--suppressions-location` option accepts a path to the file, which can be either absolute or relative to the current working directory. For example, the following command stores the suppressions in a file called `suppressions.json` in the current working directory:
+
+```
+eslint --suppress-all --suppressions-location ./suppressions.json
+```
+
+Note that you must specify the `--suppressions-location` option when running ESLint commands that use the suppressions file. For example, to run ESLint with the suppressions file from the example above:
+
+```
+eslint --suppressions-location suppressions.json
+```
+
+### Managing suppressions over time
+
+After enabling suppressions, it is recommended to commit the suppressions file to your repository. This ensures all team members have the same suppressions active when running ESLint.
+
+As you fix suppressed violations over time, the suppressions file will become outdated. ESLint outputs a warning when it finds suppressed warnings that no longer occur:
```

Comment:
```suggestion
As you fix suppressed violations over time, the suppressions file will become outdated. ESLint outputs a warning when it finds suppressed violations that no longer occur:
```

To avoid confusion as warning-level lint messages are not suppressed.